microcosm/config/api.py

```diff
@@ -19,11 +25,14 @@ def configure(registry, metadata, loader):
     Build a fresh configuration.
 
     """
-    config = Configuration({
+    defaults = {
```

Comment:
This should probably a `registry` function: `registry.get_defaults()`

Reply:
Hmm... I think I agree. Would that  mean not saving the defaults in a specific registry rather than on a magic property of the factory itself?